layouts/404.html

```diff
@@ -6,5 +6,5 @@ <h1 class="error-emoji"></h1>
     <p class="error-text">/* 404 page not found. */</p>
     <p class="error-link"><a href="/">↑ Back Home ↑</a></p>
   </div>
-  <script src="/js/bundle.js"></script>
+  <script src="{{ "/js/bundle.jaas" | relURL }}"></script>
```

Comment:
Thanks for your PR, `relURL` is better, but there's a typo here, maybe you can fix it :)

Reply:
Thank you for your responding.
I have fix it.